.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,57 @@
+---
+name: Bug Report
+description: Used to report bugs in AMP.
+labels: "Type: Bug"
+body:
+- type: markdown
+  id: header
+  attributes:
+    value: |
+      Thanks for filling out this bug report. Please only file reports about [AMP](https://amp.dev) here.
```

Comment:
"In AMP" might be ambiguous. Maybe something like:

```markdown
Please only file bug reports related to the [AMP](https://amp.dev) format and cache.

Bugs related to the AMP Wordpress plugin should be [filed on the amp-wp repository.](LINK)
```

Reply:
Good point. I've sent out #34397 with these changes.